mobile/src/lib/cozy-helper.js

```diff
@@ -73,3 +75,17 @@ export function refreshFolder (dispatch, getState) {
     }
   }
 }
+
+export const onError = (dispatch, getState, callback) => (err) => {
```

Comment:
No need for `callback`. Cf. https://github.com/cozy/cozy-files-v3/pull/179/files#r106179418